Dockerfile

```diff
@@ -0,0 +1,75 @@
+# Golang build container
+FROM golang:1.10
+
+WORKDIR $GOPATH/src/github.com/grafana/grafana
+
+COPY Gopkg.toml Gopkg.lock ./
+COPY vendor vendor
```

Comment:
You'll need to ensure that `vendor` exists in the project folder, as the building an image from a fresh git clone will fail. Maybe a `vendor/.gitkeep` file?